common-graphql/src/main/scala/queries/schema/SSO.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package queries.schemas
+
+import clue.annotation.GraphQLSchema
+
+@GraphQLSchema
+trait SSO:
+  object Scalars:
+    type UserId   = String
+    type RoleId   = String
```

Comment:
Can use use `User.Id` and `StandardRole.Id` here?

Reply:
Let me try that